### ISSUE TITLE
Vesuvio: Control position of forward and back scattering foils

### ISSUE THREAD

**Issue by kjwoodsISIS** [ISISComputingGroup/IBEX#1825]
On Vesuvio, the position of the forward and back scattering foils is critical.  As a scientist, I need to be able to continue to correctly position the foils via an OPI and via genie_python scripts.

#### Acceptance Criteria
1. I can use the IBEX GUI to select the position of the forward scattering foil.
1. I can use the IBEX GUI to select the position of the back scattering foil.
1. I can use genie_python to select the position of the forward scattering foil.
1. I can use genie_python to select the position of the back scattering foil.
1. The file(s) currently used by SECI to store the foil positions is migrated to IBEX

#### Notes
1. The position of the foils is controleld by a dual-axis McLennan controller (see [Vesuvio](https://github.com/ISISComputingGroup/IBEX/wiki/VESUVIO-Instrument-Details#noteMcLennan) )

**Comment by kjwoodsISIS** [comment]
Configure the IOC to move the motors (separate ticket - #1875)
Test that you can actually move the motors.

**Comment by AdrianPotter** [comment]
Motors set up. Need to go down to the instrument to confirm

**Comment by AdrianPotter** [comment]
Blocked by #1875 and the fault in the back scattering foil

**Comment by Tom-Willemsen** [comment]
Motors set up on Vesuvio, moving this to completed.
